vaxrank/report.py

```diff
@@ -197,11 +216,14 @@ def compute_template_data(self):
                 self.ranked_variants_with_vaccine_peptides):
             variant_short_description = variant.short_description
             if len(vaccine_peptides) == 0:
-                logger.info("Skipping %s, no vaccine peptides", variant_short_description)
+                logger.info("Skipping gene(s) %s, variant %s: no vaccine peptides",
+                    variant.gene_names, variant_short_description)
                 continue
 
             variant_data = self._variant_data(vaccine_peptides[0])
-            predicted_effect = self.variants_to_top_coding_effect_dict.get(variant)
+            # TODO(julia): is this right?
```

Comment:
@iskandr you recently added `supporting_reference_transcripts` as a property on the MutantProteinFragment, which is connected to a particular peptide. but when we display a predicted effect, it's variant-level as opposed to peptide-level. what's the best thing to do here? do you take the top peptide and grab its supporting transcripts? or do what i'm doing here now?